charts/ceph-csi-rbd/values.yaml

```diff
@@ -207,7 +207,7 @@ provisioner:
     enabled: true
     image:
       repository: k8s.gcr.io/sig-storage/csi-attacher
-      tag: v3.2.1
+      tag: v3.3.0
```

Comment:
csi-provisioner version is not updated for rbd

Reply:
updated.. thanks .ptal @Madhu-1